packages/adblocker/test/html-filtering.test.ts

```diff
@@ -212,6 +217,12 @@ describe('html-filtering', () => {
     });
+
+    it('removes matching modifiers', () => {
```

Comment:
Few things to improve here:
* HTMLSelector tests to include HTMLModifier's to ensure no breakage when both are applied
* HTMLModifier tests to cover both examples of a complete buffer but also when it is being streamed
* HTMLModifier tests to include full document tests, can be run on top of the `doc` present in the tests arleady